src/data_hub/api.py

```diff
@@ -1125,7 +1125,11 @@ def _row_to_dict(
         elif field.type in ["float", "integer", "boolean"]:
             value = field.value
         elif field.type == "select":
-            value = field.value.selected_options
+            value = field.value
+            if isinstance(value, dict):
+                value = value["selectedOptions"]
+            else:
+                value = value.selected_options
```

Comment:
this is so strange. for some reason the SDK stoppped sending objects and instead started sending dicts. no idea why

Reply:
Why does Python even need to make this distinction?